contrib/ipfs-usb/movestore

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+if [ -z "$1" ] && [ -z "$2" ]; then
```

Comment:
What is `$1` and `$2`? Assign them to properly named variables at start of file.

Reply:
There is a "syntax" section if you don't use them. But i added meaningful variables.

Reply:
Thanks